migtests/scripts/resumption.sh

```diff
@@ -41,6 +45,18 @@ fi
 source ${SCRIPTS}/yugabytedb/env.sh
 source ${SCRIPTS}/functions.sh
 
+# Handle optional additional config file ($3)
+if [ -n "$3" ]; then
+    ADDITIONAL_CONFIG_FILE="${TESTS_DIR}/resumption/additional_configs/$3"
+    if [ ! -f "${ADDITIONAL_CONFIG_FILE}" ]; then
+        echo "Error: Additional config file $3 not found in resumption/additional_configs"
+        exit 1
+    fi
```

Comment:
Just to confirm, we are not restricting the config file to be placed here under here only - `migtests/tests/resumption/additional_configs`

Few questions:
1. Is this the only one test we have as of today?
2. For eg if we add one more resumption test tomorow, will that be a different directory, what will the structure look like?

Reply:
`resumption/additional_configs` contain files which one can use to customise their runs. So the base tests will be under the resumption folder and within each test there will be a config file, or a means to generate it.
1. The files under additional_configs can be used with any of the base tests and thus giving flexibility. So think of it as one enhancement on top of the tests present.
2. If we want to add another test, it will have to be added in another directory like any test in the migtests folder.
Example test: migtests/tests/resumption/import-file/single-large-table

Reply:
Each test will have its base config which will control the test. If you wish, you can enhance that test using any additional config file present in the `resumption/additional_configs` directory. Its like an add-on to the base config.